svc-managers/mgrmodel/manager_test.go

```diff
@@ -61,7 +61,7 @@ func TestManager_Update(t *testing.T) {
 			"State": "Absent",
 		},
 	}
-	err = UpdateManagersData(key, m)
+	err = UpdateData(key, m, "Managers")
 
 	data, err := GetResource(table, key)
```

Comment:
mockGetResourcxe also needs an implementation

Reply:
Updated the mockGETResource